opentracing-redis-lettuce/src/main/java/io/opentracing/contrib/redis/spring/TracingRedisAsyncCommands.java

```diff
@@ -11,7 +11,7 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package io.opentracing.contrib.redis.lettuce;
+package io.opentracing.contrib.redis.spring;
```

Comment:
why package changed? it is under lettuce.

Reply:
mm, not sure... it's not supposed to have changed, let me revert this and the others

Reply:
sorry for that, something funny happened in my IDE and I didn't notice it